doc/user/python-tools.md

```diff
@@ -32,7 +32,7 @@ Available command line tools:
 
 - `hlb-gmy-compress`: compress an uncompressed geometry file.
 
-- `hlb-dump-extracted-properties`: convert an extraction file to CSV.
+- `hlb-dump-extracted-properties`: convert an extraction file (.xtr) to CSV. (Instead of HemeXtract, HemeXtract is obsolete for this version of HemeLB.)
```

Comment:
Rephrase: Obsolete implies HemeXtract was once current for this repo. It was developed for one of the forks.